spec/setup.js

```diff
@@ -0,0 +1,7 @@
+$(function() {
+  var jasmineEnv = jasmine.getEnv();
+  jasmineEnv.updateInterval = 250;
+
+  jasmine.getFixtures().fixturesPath =
+  jasmine.getStyleFixtures().fixturesPath = 'spec/fixtures';
+});
```

Comment:
Please add a new line at the end of each file. It might be that the Dragdealer repo is not consistent in this way, since some of the code was written a long time ago, but I've picked up this rule in the meantime. Github signals it as well: 
![image](https://cloud.githubusercontent.com/assets/250750/7785279/f64a7cfa-0191-11e5-9e6c-3d63ba628001.png)